projects/nginx/fuzz/http_request_fuzzer.cc

```diff
@@ -307,15 +307,4 @@ DEFINE_PROTO_FUZZER(const HttpProto &input) {
 
   // Will redirect to http parser
   ngx_http_init_connection(c);
-
-  // Clean-up in case of error
```

Comment:
Is this not needed anymore ?

Reply:
Yes

Reply:
Can you please explain why this is no longer required?

Reply:
The fuzzer was originally fuzzing the client side of nginx, now that it also fuzzes the server side, it handles the clean-up properly.